src/SetCookie.php

```diff
@@ -166,6 +187,10 @@ public function toHeaderValue() : string
             $headerValue .= '; httponly';
         }
 
+        if ($this->sameSite != '') {
```

Comment:
What do you think about a strict check here?

Reply:
I hadn't used one, as the type hint should prevent any type other than a string. But, feel free to change :)